Providers/Scripts/2.4x-2.5x/Scripts/nxOMSCustomLogNew.py

```diff
@@ -0,0 +1,206 @@
+#!/usr/bin/env python
+#============================================================================
+# Copyright (C) Microsoft Corporation, All rights reserved. 
+#============================================================================
+
+import os
+import imp
+import re
+import codecs
+import shutil
+import string
+import filecmp
+
+protocol = imp.load_source('protocol', '../protocol.py')
+nxDSCLog = imp.load_source('nxDSCLog', '../nxDSCLog.py')
+
+LG = nxDSCLog.DSCLog
+
+CONF_PATH = '/etc/opt/microsoft/omsagent/conf/omsagent.d/customlog.conf'
```

Comment:
This does not have to be multi-homing-compatible yet, just as an FYI to anyone else who might ask.